ui/src/types.ts

```diff
@@ -0,0 +1,23 @@
+// Defines a comment returned by the scraper.
```

Comment:
I'm not a fan of having anything in the root of src outside of the app.* files.

Reply:
Moved the offending files but it feels kind of gross with our scale of application. 

For example, we're probably never going to be doing type augmentation, but now we have an import that looks  like:
 ```typescript
import type { Filters } from '@src/types/types'
```